src/pages/stake.md

```diff
@@ -2,32 +2,32 @@
 template: stake-page
 path: /stake
 hero:
-  title: Stake
-  body: Get started staking on the Keep network.
+  title: Mint tBTC
+  body: Mint tBTC and use the first decentralized bridge between Bitcoin and
+    Ethereum.
   cta_buttons:
-    - label: Launch Dashboard
-      url: https://dashboard.keep.network/
+    - label: Mint tBTC
+      url: https://threshold.network/earn/btc
 why:
-  title: Why stake with Keep?
+  title: Why mint tBTC?
   cards:
     - title: Decentralized
-      body: Run a node on the Keep network and participate in a secure and
-        decentralized network.
+      body: tBTC is the first truly decentralized bridge between Bitcoin and Ethereum.
       icon:
         image: /images/decentralize.svg
         alt: Decentralized
     - title: Permissionless
-      body: Anyone with KEEP tokens can stake and secure the network.
+      body: Anyone can use their BTC tokens to mint tBTC.
```

Comment:
In my opinion, the word `tokens` is unnecessary in the context of Bitcoin . I would say just `BTC`. I think a `token` often refers to any cryptocurrency besides Bitcoin and Ethereum see https://www.coinbase.com/learn/crypto-basics/what-is-a-token. But I'm not sure 😄